Shared/Models/Parse Subclasses/Connection.swift

```diff
@@ -47,6 +48,11 @@ final class Connection: PFObject, PFSubclassing {
         return self.getObject(for: .conversationSid)
     }
 
+    var initialConversations: [String] {
```

Comment:
Will be used by the backend to add all new users to every conversation in this array.